src/core/manager.c

```diff
@@ -3400,7 +3403,7 @@ void manager_set_watchdog(Manager *m, WatchdogType t, usec_t timeout) {
         if (m->watchdog[t] == timeout)
                 return;
 
-        if (t == WATCHDOG_RUNTIME)
+        if (t == WATCHDOG_RUNTIME) {
```

Comment:
no `{ }` around single-line blocks, see: https://github.com/systemd/systemd/blob/main/docs/CODING_STYLE.md (applies elsewhere too)

Reply:
For this one specifically, removing the braces generates a compile warning because of the dangling else. Should I ignore the warning or keep the braces around the 'if' and the 'else if' or just keep them around the 'if'?

Reply:
if there's a compiler warning then leave it as-is